pkg/pip_requirements.txt

```diff
@@ -1,4 +1,5 @@
 spaceone-core
```

Comment:
Because of we use python-core image, When Docker image build by Dockerfile.
we do not need to add `spaceone-core` at pip_requirements.txt